pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/actions/RestartDeclarativePipelineAction.java

```diff
@@ -59,13 +60,17 @@
 import javax.servlet.ServletException;
 import java.io.IOException;
 import java.util.*;
+import java.util.logging.Logger;
 
 @ExportedBean
 public class RestartDeclarativePipelineAction implements Action {
 
+    private static final Logger LOGGER = Logger.getLogger(RestartDeclarativePipelineAction.class.getName());
+
     private final Run run;
 
     private RestartDeclarativePipelineAction(Run run) {
+        LOGGER.info("Action initiated");
```

Comment:
Probably accidentally left in from debugging: (you could also delete `LOGGER` since it looks like this was the only usage)

```suggestion
```

Reply:
Correct. I will remove it!